packages/entities/entities-vaults/src/vaults-endpoints.ts

```diff
@@ -6,7 +6,7 @@ export default {
   list: {
     konnect: {
       getAll: `${konnectBaseApiUrl}/vaults`,
-      deleteConfigStore: `${konnectConfigStoreApiUrl}/{id}`,
+      deleteConfigStore: `${konnectConfigStoreApiUrl}/{id}?force=true`,
```

Comment:
Is there a chance we may need to use this API without the `force` param?

Reply:
With the current design, no: if a config store is deleted, all its secrets become orphans and cannot be accessed anymore, so it's safe to delete them as well.